variables-and-memory/problem-set-variables-and-memory.md

```diff
@@ -1 +1,487 @@
-# Problem Set: Variables and Memory
\ No newline at end of file
+# Problem Set: Variables and Memory
+
+## Variable Scope
+
+<!-- >>>>>>>>>>>>>>>>>>>>>> BEGIN CHALLENGE >>>>>>>>>>>>>>>>>>>>>> -->
+<!-- Replace everything in square brackets [] and remove brackets  -->
+
+### !challenge
+
+* type: multiple-choice
+* id: 45bbfac6-3851-461a-9ef2-36832f43459d
+* title: Variable Scope
+<!-- * points: [1] (optional, the number of points for scoring as a checkpoint) -->
+<!-- * topics: [python, pandas] (optional the topics for analyzing points) -->
```

Comment:
For my own knowledge, what does the topic "pandas" refer to here?

Reply:
I think that refers to the python data science package `pandas`. I imagine galvanize has that in their default template since they have a data science bootcamp

Reply:
Ahh, neat. Thanks!